authorization-server/db/accesstokens.js

```diff
@@ -1,81 +1,70 @@
-/*jslint node: true */
 'use strict';
 
-//The access tokens.
-//You will use these to access your end point data through the means outlined
-//in the RFC The OAuth 2.0 Authorization Framework: Bearer Token Usage
-//(http://tools.ietf.org/html/rfc6750)
+// The access tokens.
+// You will use these to access your end point data through the means outlined
+// in the RFC The OAuth 2.0 Authorization Framework: Bearer Token Usage
+// (http://tools.ietf.org/html/rfc6750)
 
 /**
  * Tokens in-memory data structure which stores all of the access tokens
  */
-var tokens = {};
+let tokens = Object.create(null);
 
 /**
  * Returns an access token if it finds one, otherwise returns
  * null if one is not found.
- * @param key The key to the access token
- * @param done The function to call next
- * @returns The access token if found, otherwise returns null
+ * @param   {String}   key  - The key to the access token
+ * @param   {Function} done - The access token if found, otherwise returns null
+ * @returns {undefined}
  */
-exports.find = function (key, done) {
-  var token = tokens[key];
+exports.find = (key, done) => {
+  const token = tokens[key];
   return done(null, token);
 };
 
 /**
  * Saves a access token, expiration date, user id, client id, and scope.
- * @param token The access token (required)
- * @param expirationDate The expiration of the access token that is a javascript Date() object (required)
- * @param userID The user ID (required)
- * @param clientID The client ID (required)
- * @param scope The scope (optional)
- * @param done Calls this with null always
- * @returns returns this with null
+ * @param   {Object}   token          - The access token (required)
+ * @param   {Date}     expirationDate - The expiration of the access token (required)
+ * @param   {String}   userID         - The user ID (required)
+ * @param   {String}   clientID       - The client ID (required)
+ * @param   {String}   scope          - The scope (optional)
+ * @param   {Function} done           - Calls this with null always
+ * @returns {undefined}
  */
-exports.save = function (token, expirationDate, userID, clientID, scope, done) {
-  tokens[token] = {userID: userID, expirationDate: expirationDate, clientID: clientID, scope: scope};
+exports.save = (token, expirationDate, userID, clientID, scope, done) => {
+  tokens[token] = { userID, expirationDate, clientID, scope };
   return done(null);
```

Comment:
The convention for callbacks to return `done()` and not `done(null)`.  I would change this.